content/blog/deploying-cribl-stream-containers-on-hpe-greenlake.md

```diff
@@ -0,0 +1,224 @@
+REVISION: 1
+TEST SUITE: None
+```
+
+Now you can validate the deployment by running the following command:
+
+```shell
+kubectl get svc
+```
+
+You should see the following results:
+
+```shell
+NAME    	                          TYPE       	 CLUSTER-IP   	 EXTERNAL-IP  PORT(S)                                                                                                                                                    AGE
+kubernetes                         	  ClusterIP  	 10.96.0.1    	 <none>    	  443/TCP                                                                                                                                                  	 22d
+logstream-leader-1696441333        	  LoadBalancer   10.111.152.178  <pending> 	  9000:31200/TCP                                                                                                                                           	 9m56s
+logstream-leader-1696441333-internal  ClusterIP      10.105.14.164	 <none>    	  9000/TCP,4200/TCP                                                                                                                                        	 9m56s
+logstream-workergroup-1696441592   	  LoadBalancer   10.102.239.137  <pending> 	  10001:30942/TCP,9997:32609/TCP,10080:32174/TCP,10081:31898/TCP,5140:30771/TCP,8125:31937/TCP,9200:32134/TCP,8088:32016/TCP,10200:32528/TCP,10300:30836/TCP 5m35s
+```
+
+Note, the names and IP addresses will differ from the above example. To test that the deployment was successful, you can run the following command and log into your deployment using the localhost and port 9000:
+
+```shell
+kubectl port-forward service/logstream-leader-1696441333 9000:9000 &
+```
+
+#### Uninstalling Cribl using Helm
+
+You can uninstall the Cribl deployment for both the leader and worker nodes by running the following commands respectively:
+
+```shell
+helm uninstall logstream-leader-1696441333 -n default
+helm uninstall logstream-workergroup-1696441592 -n default
+```
+
+Make sure to use your leader and worker group name when uninstalling Cribl from your deployment.
+
+#### Configuring Cribl Stream
+
+Once you have deployed the Cribl Stream containers, you need to configure them to collect and process your data. You can do this by editing the Cribl Stream configuration file. The Cribl Stream documentation provides detailed instructions on how to configure Cribl Stream.
+
+#### Sending your data to your analysis platform of choice
+
+Once you have configured Cribl Stream to collect and process your data, you need to send it to your analysis platform of choice. Cribl Stream supports a wide range of analysis platforms, including Elasticsearch, Splunk, and Kafka.
+
+#### Conclusion
+
+Deploying Cribl Stream containers on HPE GreenLake is a simple and effective way to implement a vendor-agnostic observability pipeline. Cribl Stream containers offer a number of advantages, including agility, cost savings, security, and management simplicity.
```

Comment:
This type of information is actually more important to have up front. Think about it... you're trying to get a reader's attention to read your blog post. Why would they want to deploy Cribl Stream containers? You need to first explain to them why this would be a good thing.

The Conclusion (or Summary, as we often refer to it) is reserved more for a place to quickly remind them of what you've covered in the blog post, make any additional observations you might have (reitterating this is an okay thing to do), and give the readers a call to action (CTA). For instance, you could add in something like "For more information on Cribl Stream, check out ...(URL). For more blog posts related to HPE Ezmeral Software, keep coming back to the HPE Developer Community blog and search on HPE Ezmeral."